include/pokemon.h

```diff
@@ -12,16 +12,21 @@
 // Property labels for Get(Box)MonData / Set(Box)MonData
 enum {
     MON_DATA_PERSONALITY,
+    MON_DATA_STATUS,
```

Comment:
Yeah, the order relative to `MON_DATA_ENCRYPT_SEPARATOR` matters. Before means in `BoxPokemon` directly and after means in a substruct. `Pokemon` isn't encrypted, so its data could be on either side with no effect.

I'd also say that it's important to put `compressedStatus` and `hpLost` in the unencrypted data, because that preserves the performance of setting `status` and `hp` in `Pokemon`.